examples/guess-the-sketch/matchmaker/director_k8s.yaml

```diff
@@ -33,11 +33,18 @@ spec:
         - name: guess-the-sketch-director
           image: guess-the-sketch-director
           imagePullPolicy: Always
+          env:
+          - name: OM_CORE_ADDRESS
+            value: "https://om-core-976869741551.us-central1.run.app"
+          # - name: MMF_ADDRESS
+          #   value: "http://10.11.16.26"
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: fleet-allocator
+  annotations:
+    iam.gke.io/gcp-service-account: om-caller@peteryizhong-gke-dev.iam.gserviceaccount.com
```

Comment:
Shall we create a service account dynamically in the user's project when a user setting up the demo?

Reply:
I don't really want to modify the terraform files at top of the GenAI-quickstart repo to create this Google service account because it's pretty specific to this demo, and only if the customer wants to deploy OM2 on Cloud Run.

As a remedy, I've changed the service account to be `YOUR_GOOGLE_SERVICE_ACCOUNT` and modified the README.md of the Guess-the-Sketch demo to include the step where they should create this Google service account.